tests/prefetch_related/test_prefetch_related_objects.py

```diff
@@ -32,6 +34,34 @@ def setUpTestData(cls):
         cls.reader1.books_read.add(cls.book1, cls.book4)
         cls.reader2.books_read.add(cls.book2, cls.book4)
 
+    def test_add_clears_prefetched_objects(self):
```

Comment:
Considering the docstring of this test class, I would put the test in `tests.py`.